services/src/main/java/org/keycloak/theme/ClasspathThemeProviderFactory.java

```diff
@@ -36,6 +36,10 @@
 
     private String id;
 
+    public ClasspathThemeProviderFactory() {
```

Comment:
This ctor seems to be necessary for quarkus to avoid:
```
RROR: Failed to start server using profile (dev)
ERROR: java.lang.NoSuchMethodException: org.keycloak.theme.ClasspathThemeProviderFactory.<init>()
ERROR: org.keycloak.theme.ClasspathThemeProviderFactory.<init>()
For more details run the same command passing the '--verbose' option. Also you can use '--help' to see the details about the usage of the particular command.
```